src/LanguageServer.cpp

```diff
@@ -504,25 +504,6 @@ void LanguageServer::onInitialized([[maybe_unused]] const lsp::InitializedParams
         client->registerCapability("didChangeConfigurationCapability", "workspace/didChangeConfiguration", nullptr);
     }
 
-    // Dynamically register file watchers
-    if (client->capabilities.workspace && client->capabilities.workspace->didChangeWatchedFiles &&
-        client->capabilities.workspace->didChangeWatchedFiles->dynamicRegistration)
-    {
-        client->sendLogMessage(lsp::MessageType::Info, "registering didChangedWatchedFiles capability");
-
-        std::vector<lsp::FileSystemWatcher> watchers{};
-        watchers.push_back(lsp::FileSystemWatcher{"**/.luaurc"});
-        watchers.push_back(lsp::FileSystemWatcher{"**/sourcemap.json"});
```

Comment:
We should only move the registration of `sourcemap.json` to the roblox platform, and leave everything else the same. I think it should be fine to register multiple watcher capabilities with different IDs.

Reply:
Are RobloxLanguageServer and LanguageServer somehow separate with regards to capabilities? Sorry I don't really know what you mean by IDs. However, I did push changes separates the watchers so that only sourcemap.json is in the roblox platform.

Reply:
I checked the response it was getting an error. Looks like it was from a typo.